doc/user/tutorials/interacting-with-geospatial-data.md

```diff
@@ -200,17 +232,10 @@ view by clicking on the gear icon in the **Attributes Toolbar** or by going to
 For more in-depth information about the Processing Toolbox see the 
 [QGIS User Manual](https://docs.qgis.org/3.28/en/docs/user_manual/processing/toolbox.html)
 
-## Spatial Querying
-
-Spatial querying allows the user to select specific layer features based on desired parameters,
-or compare features from one layer with features from another layer based on their
-spatial relationships or common parameters. Below we describe a specific set of steps for
-completing various example query and analysis tasks; however, the QGreenland user will
-learn through experience that there is very often more than one way to complete a desired
-task in the QGIS platform. We will describe several query and analysis methodologies that
-use the QGIS **Processing Toolbox**.
```

Comment:
Yes, the examples are not specific to querying and the "Processing Toolbox" section that the examples become a part of (just before this text) serves as introductory text. 

I decided not to keep the "there is more than one way" text because there's a fair bit of content _before_ this paragraph that also describes multiple ways of doing things. Maybe a note at the top? Is it really that helpful to point this out?

Reply:
Moved note to top of tutorial page.